client/client_test.go

```diff
@@ -56,10 +54,10 @@ func setupCloudproberAndClient(t *testing.T) (context.Context, *hermes.Hermes, *
 		t.Fatalf("Cloudprober gRPC Client could not be initialised: %v", err)
 	}
 
-	return context.Background(), testHermes, client
+	return context.Background(), client
```

Comment:
What do we use this context for? Could we just not return it?

Reply:
The context is used in all of the client method calls, but I've refactored this so that we create the context in the test and pass it to this method, rather than receiving it as a return value.